src/App.vue

```diff
@@ -24,7 +24,7 @@ provide('notify', notify);
 
 const space = computed(() => {
   const key = aliases[domain] || domain || route.params.key;
-  return explore.value.spaces?.[key] ? explore.value.spaces[key] : {};
+  return explore.value.spaces?.find(s => s.id === key) ?? {};
```

Comment:
What's. the need for find here? The id is always the key, we don't need to iterate over that object.

Reply:
I change that, explore.spaces is now an array of objects. The same as the response from graphql spaces query.

Reply:
How it help to have it as array?

Reply:
I thought it would help, but in the end it didn't really make any difference and I didn't see a good reason for changing it back.

Reply:
@bonustrack after having a chat with Harsh I now see your point, I'll make those changes now.